classes/converter_chromium.php

```diff
@@ -103,7 +103,8 @@ protected function generate_pdf_content(moodle_url $proxyurl, string $filename =
             $page->navigate($proxyurl->out(false))->waitForNavigation();
             $pdf = $page->pdf($options);
 
-            return base64_decode($pdf->getBase64());
+            $timeout = 1000 * helper::get_config($this->get_name() . 'responsetimeout');
```

Comment:
Should this have `chromiumresponsetimeout` as well @dmitriim ?
```suggestion
            $timeout = 1000 * helper::get_config($this->get_name() . 'chromiumresponsetimeout');
```

Reply:
Yeah. Look like should be responsetimeout and chromium will be prefixed in the helper.